playbooks/roles/ses-rook/tasks/main.yml

```diff
@@ -89,6 +89,37 @@
   retries: 30
   delay: 15
 
+# kubectl cp cannot be use due to missing tar in the container
+- name: Get mons from ceph.conf of the ceph tools container
+  shell: |
+    set -o pipefail
+    kubectl -n rook-ceph exec $(kubectl -n rook-ceph get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}') -- \
+    sed -n 's/mon_host = //p' /etc/ceph/ceph.conf
+  register: _rook_ceph_conf_mon_host
+
+- name: Generate ses-config.yml
+  delegate_to: localhost
+  template:
+    src: "ses-config.yml.j2"
+    dest: "{{ socok8s_ses_pools_details }}"
+
+- name: Find the admin keyring b64 value
+  shell: |
+    set -o pipefail
+    kubectl -n rook-ceph exec $(kubectl -n rook-ceph get pod -l 'app=rook-ceph-tools' -o jsonpath='{.items[0].metadata.name}') \
+    ceph auth get-key client.admin | base64
+  changed_when: false
+  register: _clientadminkey
+
+- name: Create user_variables
+  delegate_to: localhost
+  blockinfile:
+    path: "{{ socok8s_extravars }}"
+    create: yes
+    block: |
+      ceph_admin_keyring_b64key: {{ _clientadminkey.stdout_lines[0] }}
+      ceph_user_keyring_b64key: {{ _clientadminkey.stdout_lines[0]  }}
+      # not adding suse_osh_deploy_ceph_mons - relying on ceph.conf values
```

Comment:
I dont understand this one? Dont we need those as per the template below?

Reply:
Oh, I see, it gets them from the var directly...wow, I did not expec that to work. So the template can access all vars in the namespace of course, but using that "temporal" one feels a bit wrong.

Reply:
The original code (see `common-ses/tasks/ses_config.yml`) replaces the mon values with the loop over `ansible_all_ipv4_addresses`. I don't think this is needed, the values from ceph.conf should be usable in `deploy-osh` role..